mozregression/cli.py

```diff
@@ -285,7 +285,7 @@ def create_parser(defaults):
             "x86_64",
         ),
         default=None,
-        help=("Force alternate build (applies to GVE, Fenix, and Focus)."),
+        help=("Force alternate build (applies to Firefox, GVE, Fenix, and Focus)."),
```

Comment:
For completeness might be good to add `Firefox-l10n` to the list as well, since that is technically treated as a separate app here.

Reply:
```suggestion
        help=("Force alternate build (applies to Firefox, Firefox-l10n, GVE, Fenix, and Focus)."),
```